### PR TITLE
Initial modal

### LINKED ISSUE #108: Create a modal dialog for adding service
Create a modal view for adding a service to the portal.

Definition of done
---
* [ ] Modal dialog template exists.
* [ ] Optionally, modal dialog template takes arguments or data context for organization id and department id.
* [ ] Modal dialog contains autoform.
* [ ] Autoform contains most or all service related fields.

### PR DESCRIPTION
**Pull request by elnzv**
Closes #108

# Proposed changes
- modal to add new service

### CODE REVIEW

**Comment by brylie** [comment]
@elnzv what is the status of this issue?

**Comment by elnzv** [comment]
@Digipalvelutehdas/dipor-team, please review.

**Comment by brylie** [comment]
Looks good. Will merge after #151 gets merged.

**Comment by brylie** [comment]
@elnzv there are a few conflicts that need to be merged here.

File: departments/collection/helpers.js [hunk 1]
```diff
@@ -12,5 +12,14 @@ Departments.helpers({
     } catch(err) {
       return false;
     }
+  },
+  "services" () {
+    // Assign department ID instance to a variable
+    const departmentId = this._id;
+
+    // Find all services related to current department
+    const departmentServices = Services.find({departmentId}).fetch();
```
Comment (brylie) [inline departments/collection/helpers.js]: We may not need the `fetch()` call here. Try removing `.fetch()`, so this helper returns a database cursor.

**Pull request merged**